projects/packages/search/src/customberg/components/sidebar/sidebar-options.jsx

```diff
@@ -96,16 +96,12 @@ export default function SidebarOptions() {
 					value={ trigger }
 					options={ [
 						{
-							label: __( 'Open when the user starts typing', 'jetpack-search-pkg' ),
-							value: 'immediate',
-						},
-						{
-							label: __( 'Open when results are available', 'jetpack-search-pkg' ),
-							value: 'results',
+							label: __( 'Open when user submits the form (recommended)', 'jetpack-search-pkg' ),
```

Comment:
This is minor but when user has `results` as the previous trigger, user would see `submit` as their choice in Customizer and Customberg. Should we add a mapping from filtering the option? By doing so, we also would only need to remove [the logic](https://github.com/Automattic/jetpack/pull/25393/files#diff-43f0aaae6d0a58f8268ff808c08906a16fb7c25bf783dc267b4986b6c1d9d298L145) for `results` on the frontend.

Reply:
> By doing so, we also would only need to remove the logic for results on the frontend.

Possibly I'm wrong about this part. Actually it might be better to keep this for robustness, just in case we remove the compatibility patch later.

Reply:
This sounds like a good idea to me 👍 

Do you think the mapping from 'results' > 'immediate' makes sense? I would rather move people on to 'submit' really but it seemed the lowest impact option.

Reply:
> Do you think the mapping from 'results' > 'immediate' makes sense?

Yep probably results is the lowest impact option - less surprise would be better IMO, because for site with `results` as trigger might feel broken if we map it to `submit`. I doubt there are many sites using the option tho.

Reply:
I added a compatibility patch before just FYI: https://github.com/Automattic/jetpack/blob/trunk/projects/packages/search/compatibility/search-0.15.2.php

And we'll need to load the fix to WPCOM explicitly if the fix falls in the `compatibility` folder for now. I'll propose a fix to automatically load soon.